mypy/waiter.py

```diff
@@ -62,7 +62,7 @@ def status(self) -> Optional[int]:
     def read_output(self) -> str:
         with open(self.outfile.name, 'rb') as file:
             # Assume it's ascii to avoid unicode headaches (and portability issues).
-            return file.read().decode('ascii')
+            return file.read().decode('utf-8')
```

Comment:
Probably I had mystery Unicode characters in my test somewhere. If I run the test-suite (with my test added), it raised and Exception about being unable to convert to unicode. I should go over my tests and fix the root error instead there